app/models/physical_storage_family.rb

```diff
@@ -0,0 +1,19 @@
+class PhysicalStorageFamily < ApplicationRecord
+  # include_concern 'Operations'
+
+  include NewWithTypeStiMixin
+  include ProviderObjectMixin
+  include AvailabilityMixin
+  include SupportsFeatureMixin
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id
+  has_many :physical_storages, :dependent => :nullify
+
+  acts_as_miq_taggable
+
+  def self.class_by_ems(ext_management_system)
```

Comment:
This is typically used by the UI for the purpose of checking for supported features on the model, is the UI going to be doing operations on `PhysicalStorageFamily` objects?  I thought this was going to be used to populate a drop down for "available storage families".

Reply:
We planned on eventually making a GUI of storage-families, allowing administrators to view and update them. But that's not happening soon. And we can do the needed changes when we add the GUI.